app-catalog/src/components/charts/List.tsx

```diff
@@ -21,13 +22,23 @@ import { Autocomplete, Pagination } from '@mui/material';
 import { useEffect, useState } from 'react';
 //import { jsonToYAML, yamlToJSON } from '../../helpers';
 import { fetchChartsFromArtifact } from '../../api/charts';
-import CNCFLight from './cncf-icon-color.svg';
+import { AppCatalogTitle } from './AppCatalogTitle';
 //import { createRelease } from '../../api/releases';
 import { EditorDialog } from './EditorDialog';
 
+interface AppCatalogConfig {
+  /**
+   * Show only verified packages. If set to false shows all the packages
+   */
+  showOnlyVerified: boolean;
+}
+
+export const store = new ConfigStore<AppCatalogConfig>('app-catalog');
+const useStoreConfig = store.useConfig();
+
 export const PAGE_OFFSET_COUNT_FOR_CHARTS = 9;
 
-export function ChartsList({ fetchCharts = fetchChartsFromArtifact }) {
+export function ChartsList({ fetchCharts = fetchChartsFromArtifact }, props) {
```

Comment:
React components only have one parameter. That second `props` parameter looks wrong

Reply:
ah this was left from experimenting with storybook